src/api/rif-notifier-service/subscriptionBatch.ts

```diff
@@ -0,0 +1,23 @@
+import { NotifierAPIService, NotifierResponse } from './interfaces'
+import { SubscribeToPlanDTO, SubscribeToPlanResponseDTO } from './models/subscriptions'
+
+export const address = 'subscribeToPlan' as const
+export default class SubscribeToPlanService
```

Comment:
```suggestion
export const address = 'subscribeToPlan' as const
type SubscribeToPlanResponse = NotifierResponse<SubscribeToPlanResponseDTO>

export default class SubscribeToPlanService
```

to go with the change to `_create`